### ISSUE TITLE
[FEATURE REQUEST] A sprint TODO view

### ISSUE THREAD

**Issue by pearupung** [lfos/calcurse#181]

**Comment by lhca** [comment]
I agree. The TODO panel need better ways for selecting or limiting what is on display: marking items or searching for items or ...

I am thinking about it.
